NEWS

```diff
@@ -2,6 +2,10 @@ systemd System and Service Manager
 
 CHANGES WITH 243 in spe:
 
+        * Enable ping(8) inside rootless Podman containers by turning on the
```

Comment:
please drop the reference to Podman, this is in no way specific to Podman... Also the term "rootless" is bogus. podman requires suid binaries (/usr/sbin/newuidmap), to operate. It's still needs root, just with. I mean, I am can ignore this if the podman project is a bit misleading on its own documentation, but I do object advertising this in our systemd NEWS file, please!

Reply:
Ok, reworded.